arches/install/arches-templates/.github/workflows/main.yml

```diff
@@ -61,9 +61,12 @@ jobs:
             echo "package.json not found, skipping npm commands."
           fi
 
-      - name: Check formatting
+      - name: Check frontend formatting with prettier
         run: |
           npm run prettier:check
+
+      - name: Check backend formatting with black
+        run: |
           black . --check --exclude=node_modules
```

Comment:
Separated these as we got multiple reports of users finding the concat'ed logging of prettier and black confusing.

The other approach here would be to just run the pre-commit hooks in CI, but we have some overlap with the webpack build step there. I think this works for now, but we can reevaluate if we add other hooks.